labelbox/data/annotation_types/data/raster.py

```diff
@@ -20,6 +21,7 @@ class RasterData(BaseModel, ABC):
     file_path: Optional[str] = None
     url: Optional[str] = None
     arr: Optional[TypedArray[Literal['uint8']]] = None
+    cache_data: Optional[TempFile] = None
```

Comment:
Note: this is here so that a reference is persisted when creating the temp file. This is so that garbage collection doesn't immediately remove the file and relevant file path

Reply:
This isn't necessary. If you maintain a reference to this object and set `self.cache_data = ...` then this won't get gc-ed.